src/utilities/index.js

```diff
@@ -60,3 +60,13 @@ export {
 
 // Asserts that a string is a valid GraphQL name
 export { assertValidName } from './assertValidName';
+
+// Helpful functions to compare two GraphQLSchemas and detect breaking changes.
+export {
+  findBreakingChanges,
```

Comment:
You probably only need to export this function (same below). All the other functions are really just implementation details of this one.